ios/Classes/BetterPlayer.m

```diff
@@ -924,6 +924,28 @@ - (void) hideLimitedPlanCoverViewInPIP {
     }
 }
 
+/// Hide LimitedPlanCoverView after the PIP is completely gone.
+///
+/// - if _isPipMode == true and isPictureInPictureActive is whatever (still in PIP mode)
+///     - All the check is false, finish this function without any actions.
+///     - The showLimitedPlanCoverView logics already handled when re-entering PIP and while in PIP mode.
+/// - if _isPipMode == false and isPictureInPictureActive == true (while exiting PIP, and not completed yet)
+///     - Call the loop and re-checking after 0.1 seconds.
+/// - if _isPipMode == false and isPictureInPictureActive == false (the PIP is completely gone)
+///     - Hide LimitedPlanCoverView.
+- (void)hideLimitedPlanCoverAfterPipCompletelyGone {
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.1 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
```

Comment:
- Memo:
With `0,1`s delay, in most of the cases, the function is only called one time without any loop.